### PR TITLE
router: Various improvements and fixes

### LINKED ISSUE #7: [router] Updates to various configs do not trigger reloads/restarts
- [x] bgpd.conf
- [x] dhcpd.conf
- [x] rtadvd.conf
- [ ] sysctl.conf

The hostname.* files are tricky because I don't think we necessarily just want to issue /etc/netstart. But maybe that's safe. ¯\\\_(ツ)\_/¯

### PR DESCRIPTION
**Pull request by iliana**
Closes #7, #8, #9, #10, #11, #12, and additionally:

* Makes the autoinstaller easier to run when testing changes to roles by allowing users to specify a repo URL and branch from which to run roles
* Reorders steps in the autoinstaller; there is now a clear line where network access (nearly) stops when running the roles by deleting /etc/resolv.conf (something we have to do anyway because of Routing Problems We Don't Understand In Production)
* `&&` everywhere in autoinstaller SSH commands
* Speeds up playbook execution by not checking for openbsd_pkg updates
* Fixes a daily insecurity output message by explicitly setting our user hashed passwords to `*************`

These changes are currently live and I've tested the autoinstaller successfully installs.

### CODE REVIEW

File: sea1/router/files/bgpd.conf
Comment (euank) [inline sea1/router/files/bgpd.conf:292]: o7

File: sea1/router/playbook.yaml
Comment (euank) [inline sea1/router/playbook.yaml:17]: That also means things above this line can only be changed if this role is used against the build_vm thingy, not the real router.

For my own clarification: the intent is that this role can both provision new disks (in the case of the vm thing), and update configuration on the live router (e.g. for bgpd changes). That's right, yes?

I think the readme could possibly point out that distinction, and this line can as well...

**Review by euank — APPROVED** [review]
fancy, I like the shiny new notify handlers.

This LGTM, assuming my understanding is correct in the for-clarification comment. If you want to add some additional notes of clarification before merging, that works too.

File: sea1/router/playbook.yaml
Comment (iliana) [reply sea1/router/playbook.yaml:17]: Yes, your interpretation is right. It should probably say something closer to "tasks hitting the internet while bootstrapping must go above this line". Opened #14 :)

**Pull request merged**
